### DISCUSSION TITLE
在透明代理/系统代理实现方式中没有redirect选项

### DISCUSSION THREAD

**Discussion by KeepingRunning** [v2rayA/v2rayA#1539] [Q&A] [answered]
所用系统：
Distributor ID:    Ubuntu
Description:    Ubuntu 24.04.1 LTS
Release:    24.04
Codename:    noble

当我想更改设置时，我发现透明代理中只有“system proxy”选项，不能选择”redirect“，并且当我尝试使用“system proxy”时，会有提示信息“failed to start v2ray-core: not support "system proxy" mode of transparent proxy: does not support to configure system proxy on your OS”

设置界面如下所示：
![settings](https://github.com/user-attachments/assets/4b0149ca-6752-47ef-9941-2dcca215656a)

请问该如何解决

**Comment by BIDBUG** [comment, marked as answer]
不要执行v2raya --lite，执行sudo v2raya有别的选项可以选择redirect。v2raya --lite貌似是windows系统才用的。

**Comment by Days-gone** [comment]
@KeepingRunning , hi, i meet the same problem as you, how did you solve this problem?
i tried to boot up v2raya, using the follow two ways:
```
sudo systemctl start v2raya.service
```
and also try to open the v2raya in the terminal
```
sudo v2raya
```
both ways have failed to solve the problem, i still failed to turn on the system proxy on ubuntu2404, debug message:
```
invalid config: not support "system proxy" mode of transparent proxy: does not support to configure system proxy on your OS
```

**Comment by BIDBUG** [comment]
<img width="777" height="556" alt="截图 2026-02-01 10-04-41" src="https://github.com/user-attachments/assets/df05bd7f-6e2c-4a6c-9837-a818f9cb6a3f" />

`v2raya`不止一种代理方式，`system proxy`只有`windows`才能用，其他代理方式也有些区别，`redirect`可能无法通过限制ip的网页比如`chatgpt`，可以改为`tproxy`代理方式。其他代理方式你可以自己尝试一下。使用`v2raya --lite`进入只有`system proxy`代理方式，`sudo v2raya`或者
`sudo systemctl start v2raya.service`才有其他代理方式

`v2raya` offers more than one proxy method. The `system proxy` can only be used on `Windows`, while other proxy methods also differ to some extent. The `redirect` method may not work on websites with IP restrictions, such as `ChatGPT`, and can be switched to the `tproxy` proxy method. You can try out other proxy methods yourself. Using `v2raya --lite` will only provide the `system proxy` method, while `sudo v2raya` or `sudo systemctl start v2raya.service` will include other proxy methods.
